gstd/gstd_pipeline.c

```diff
@@ -398,6 +417,18 @@ gstd_pipeline_set_property (GObject * object,
       }
       self->state = g_value_get_object (value);
       break;
+    case PROP_VERBOSE:
+
+      self->verbose = g_value_get_boolean (value);
+      if(self->verbose == TRUE){
+        deep_notify_id =
+          gst_element_add_property_deep_notify_watch (self->pipeline, NULL, TRUE);
+      }
+      if(self->verbose == FALSE && deep_notify_id !=0 ){
```

Comment:
If you use the id only, you can simplify to a regular ìf/else` statement. Also, this code is poorly indented.